### PR TITLE
Update train_resnet_p2.py

### PR DESCRIPTION
**Pull request by GRP2019**
line174 running_loss += loss.data.item()

### CODE REVIEW

**Comment by youngminPIL** [comment]
Thank you for your feedback, I have fixed my code.
